examples/kubeflow_pipelines_orchestration/README.md

```diff
@@ -239,8 +297,39 @@ zenml stack register gcp_kubeflow_stack \
     -c gcr_registry \
     --set
 
-# Forward the Kubeflow pipelines UI so we can access it locally
-zenml stack up
+```
+
+### Setting up the stack using a recipe
+
+Alternatively, you can also use the `zenml stack recipe` command to create the
+stack and its components. Just as the local k3d stack, the GCP Kubeflow stack
+recipe is also available in the ZenML repository. You can pull it using the
+following command:
+
+```bash
+zenml stack recipe pull gcp_kubeflow_stack
+```
+
+You can follow the same steps as above to create and activate the stack and its
+components:
+
+Deploy the stack using the ZenML CLI:
+
+```shell
+zenml stack recipe deploy gcp_kubeflow_stack
+```
+
+> **Note**
+    > This command can also automatically import the resources created as a ZenML stack for you. Just run it with the `--import` flag and optionally provide a `--stack-name` and you're set! Keep in mind, in that case, you'll need all integrations for this example installed before you run this command.
+
+
+You can run the following command to import the resources as a ZenML stack, manually.
+
+```shell
+zenml stack import <STACK_NAME> -f <PATH_TO_THE_CREATED_STACK_CONFIG_YAML>
+
+# set the imported stack as the active stack
+zenml stack set <STACK_NAME>
```

Comment:
Same here, wouldn't this be much simpler? Wouldn't even mention the alternative IMO

```suggestion
```shell
zenml stack recipe deploy gcp_kubeflow_stack --import --stack-name <STACK_NAME>
```
```